### PR TITLE
Pipe switch

### PR DESCRIPTION
**Pull request by npaterno**
Change %>% to |> throughout (Issue #168 ) including images in Tutorial 1 Lesson 1

### CODE REVIEW

File: 06-model-infer/01-lesson/06-01-lesson.Rmd [hunk 1]
```diff
@@ -161,9 +161,9 @@ Indeed, when we take repeated samples of size 20 (here we took 50 different samp
 
 
 ```{r echo = FALSE}
-starbucks_many_lm <- starbucks_many %>% 
-  group_by(replicate) %>% 
-  do(tidy(lm(Calories ~ Fat, data=.))) %>%
+starbucks_many_lm <- starbucks_many |> 
+  group_by(replicate) |> 
+  do(tidy(lm(Calories ~ Fat, data=.))) |>
```
Comment (mine-cetinkaya-rundel) [inline 06-model-infer/01-lesson/06-01-lesson.Rmd:166]: Does this work as expected?

File: 06-model-infer/01-lesson/06-01-lesson.Rmd [hunk 2]
```diff
@@ -209,9 +209,9 @@ The same analysis can be done with different variables.  Here, consider fat and
 ### Fat & carbohydrates: sampling distribution of slopes
 
 ```{r echo = FALSE}
-starbucks_many_lm <- starbucks_many %>% 
-  group_by(replicate) %>% 
-  do(tidy(lm(Carbs ~ Fat, data=.))) %>%
+starbucks_many_lm <- starbucks_many |> 
+  group_by(replicate) |> 
+  do(tidy(lm(Carbs ~ Fat, data=.))) |>
```
Comment (mine-cetinkaya-rundel) [inline 06-model-infer/01-lesson/06-01-lesson.Rmd:214]: Similarly this?

**Review by mine-cetinkaya-rundel — APPROVED** [review]
LGTM! I left two questions where `.` is used in the pipeline to make sure it's working as expected. If so, good to merge!

File: 06-model-infer/01-lesson/06-01-lesson.Rmd (hunk 1 above)
Comment (npaterno) [reply 06-model-infer/01-lesson/06-01-lesson.Rmd:166]: It does. I ran all code chunks and compared to the online tutorial and output is the same.

File: 06-model-infer/01-lesson/06-01-lesson.Rmd (hunk 2 above)
Comment (npaterno) [reply 06-model-infer/01-lesson/06-01-lesson.Rmd:214]: Yep :)

**Pull request merged**